### DISCUSSION TITLE
Help with consume folder

### DISCUSSION THREAD

**Discussion by flexxipanda** [jonaswinkler/paperless-ng#1216] [Support] [answered]
Hi guys,

I need some help with the setup. I'm quite noob at ubuntu/linux and this the first time I work with ubuntu and docker. I've setup pihole and OMV before on raspis though. 

Anyway I installed docker and docker-compose and ran the installation script 

> "curl -L https://raw.githubusercontent.com/jonaswinkler/paperless-ng/master/install-paperless-ng.sh | bash"

And I'm at the step where you specify your "target" and "consume" folder.

I have a "dumb" NAS at home which has SMB drives (Openmediavault on Raspi).

I tried to set the consume folder like this: smb://raspberrypi2/scan

So I would Scan to the folder on the network drive and paperless looks into that. But it tells me it is an invalid folder. Is it possible at all to do it like this? Or am I doing something wrong ? (As I said I'm noob at linux, but learning!)

Also I tried to install it with mostly default settings and I got an Error at the end:

> docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', PermissionError(13, 'Permission denied'))
> [64806] Failed to execute script docker-compose

Here is the full log: https://pastebin.com/gdRLd1Jj

**Comment by steilerDev** [comment, marked as answer]
Hi @flexxipanda, the docker can (afaik) only access the local filesystem on the system. If paperless is not running on the NAS you would need to mount the directories first into your filesystem (something like this: https://support.zadarastorage.com/hc/en-us/articles/213024986-How-to-Mount-a-SMB-Share-in-Ubuntu) and specify the mount point. I hope this makes sense :)

**Reply by flexxipanda** [reply]
Thanks man that helped a lot!
